CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ UNRELEASED
 ----------
 
 * [ [#951](https://github.com/digitalfabrik/integreat-cms/issues/951) ] Add possibility to create categories for POIs
+* [ [#1703](https://github.com/digitalfabrik/integreat-cms/issues/1703) ] Removed excessive message print when creating a user whose name already exists
```

Comment:
```suggestion
* [ [#1703](https://github.com/digitalfabrik/integreat-cms/issues/1703) ] Remove pending account activation warning when user form is submitted with errors
```
Sorry for being nitpicky here :grin: 
Two small ideas for improvement:
- I think we can be more precise about what has really been changed
- We try to use the [imperative mood](https://cbea.ms/git-commit/#imperative) for commit messages and changelog entries